core/modules/file.py

```diff
@@ -803,3 +805,15 @@ def doCleanupDeletedFiles(cursor=None):
     newCursor = query.getCursor()
     if newCursor:
         doCleanupDeletedFiles(newCursor)
+
+
+@PeriodicTask(60 * 4)
+def start_delete_pending_files():
+    """
+    Start deletion of pending FileSkels that are older than 7 days.
+    """
+    DeleteEntitiesIter.startIterOnQuery(
+        FileBaseSkel().all()
+        .filter("pending =", True)
+        .filter("creationdate <", datetime.now() - timedelta(days=7))
```

Comment:
I'm not sure, but shall be use `utcNow()` here, as `creationdate` is a timezone-aware value?
```suggestion
        .filter("creationdate <", utils.utcNow() - timedelta(days=7))
```